ezcontent.libraries.yml

```diff
@@ -0,0 +1,10 @@
+ezcontentjs:
+  version: 1.x
+  js:
+      js/ezcontent.js: {}
+  
+  dependencies:
+    - core/drupal
```

Comment:
why we need other dependencies here, i believe only required dependency is `core/jquery`